packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -672,6 +672,30 @@ export const globalOptions = {
       }
     },
   }),
+  accountSelectWithAll: createOption({
+    key: 'account' as const,
+    prompt: account.accountSelectAllPrompt,
+    defaultIsOptional: false,
+    validation: z.string(),
+    option: new Option(
+      '-a, --account-alias <account>',
+      'Enter your account alias file',
+    ),
+    expand: async (
+      accountAlias: string,
+    ): Promise<IAliasAccountData | undefined> => {
+      try {
+        if (accountAlias === 'all') {
+          return;
+        }
+
+        const accountDetails = await readAccountFromFile(accountAlias);
+        return accountDetails;
+      } catch (error) {
+        throw new Error(error.message);
```

Comment:
@barthuijgen we probably shouldnt throw errors from the createOptions, it happens in 5 more places toughts

Reply:
Now that I'm looking at it more, especially in `expand` I would not want to see an error, the return type is already `| undefined` so that makes it even weirder.

Reply:
agreed, we will make a new ticket to remove the throwing of errors from the prompts